react/week4/githubuser/src/GithubRepos.js

```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from "react";
+import { UsersContext } from "./UsersContext";
+
+const GitHubRepos = () => {
+  const repoContext = useContext(UsersContext);
+  //fetching the github repos
+  const getGitHubRepos = () => {
+    fetch(`https://api.github.com/users/HackYourFuture-CPH/repos`)
+      .then((res) => res.json())
+      .then((reposdata) => {
+        repoContext.setRepos(reposdata);
+      });
+  };
```

Comment:
You have the `getGitHubUserWithFetch` defined in the context, but this one is this component.
I think either way works, but you should choose one of them :)